hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -201,10 +206,10 @@ private Blockchain stubbedBlockchain() {
 	private ProcessableBlockHeader stubbedBlockHeader(long timestamp) {
 		return new ProcessableBlockHeader(
 				Hash.EMPTY,
-				Address.ZERO, // TODO Coinbase might be the 0.98 address
+				Address.fromHexString(asSolidityAddressHex(txnCtx.submittingNodeAccount())),
 				Difficulty.ONE,
 				0,
-				12_500_000L, // TODO GlobalDynamic property maxGas
+				properties.maxGas(),
```

Comment:
Same comment here. You have to modify `long gasLimit = 15000000L;` on line 163 as-well